test/await-sef.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const { test } = require('tap')
+const boot = require('..')
+
+test('await self', async (t) => {
+  const app = {}
+  boot(app)
+
+  t.is(await app, app)
```

Comment:
What happens if you call `await` app multiple times?
```suggestion
  t.is(await app, app)
  t.is(await app, app)
  t.is(await app, app)
```

Reply:
It all works fine.